Sources/Commands/Completions+zsh.swift

```diff
@@ -12,6 +12,9 @@ import Foundation
 import Basic
 import Utility
 
+let dependencies = "swift package show-dependencies | grep -oE '── (.*)<' | sed -E 's/── (.*)</\\1/'"
+let executables = "swift package describe | grep 'Type: executable' -B 3 | grep 'Name:' | sed -e 's/    Name: //'"
```

Comment:
Instead of shelling out to get this data, we can directly get it from the package graph object.